src/main/java/uk/gov/hmcts/divorce/document/content/DocmosisCommonContent.java

```diff
@@ -48,6 +51,9 @@ public class DocmosisCommonContent {
     @Value("${court.locations.serviceCentre.poBox}")
     private String poBox;
 
+    @Value("${court.locations.serviceCentre.poBoxCy:Blwch Post 13226}")
```

Comment:
does it need a default ? just thinking from consistency point as `@Value("${court.locations.serviceCentre.poBox}")
     private String poBox;` doesn't has default

Reply:
we can add this in app config if we don't want to mention it here as a default?